src/components/repoSelector.js

```diff
@@ -0,0 +1,55 @@
+import ReactDiffViewer, { DiffMethod } from "react-diff-viewer";
+import React, { useEffect, useState } from "react";
+import { useAllOrgs, useOrgRepos } from "dcs-react-hooks";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+export default function Repository(orgName) {
```

Comment:
I would pass two props in here as well just like my comment on the other component. This will allow the values to be used outside of this component.

```json
{
  organization, // this is the string from the app state, same as your use of orgName
  repository, // this is the string from the app state
  onRepository, // this is the callback to update the app state of repository
}
```